gbpservice/nfp/config_orchestrator/common/lbv2_constants.py

```diff
@@ -0,0 +1,18 @@
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
```

Comment:
@njagadish Those constants only used in notification handler, do you recommend me to move them to the beginning of gbpservice/nfp/config_orchestrator/handlers/notification/handler.py ?